Amazon.IonDotnet/Builders/IonTextOptions.cs

```diff
@@ -20,6 +20,17 @@ namespace Amazon.IonDotnet.Builders
     public class IonTextOptions
     {
         public static readonly IonTextOptions Default = new IonTextOptions();
+        public static readonly IonTextOptions Json = new IonTextOptions
+        {
+            JsonDowngrade = true,
```

Comment:
Should we emulate ion-java and not have a JsonDowngrade flags?

I think you'd need to add `DecimalAsFloat`, `ClobAsString`, and `SexpAsList` (the latter of which just trigger exceptions at this point).

Reply:
I don't think I necessarily agree here. It's probably better to add a //TODO on the JsonDowngrade flag (it's internal so customers can't see it anyways) and eventually remove it when and if everything is implemented.

The reason I disagree is because certain attributes I think are quite odd and don't make sense to expose, such as InfNanFloatsAsNull and then DecimalAsFloat (which decimal? Ion Decimal or C# decimal?). It's just somewhat ambiguous and not very helpful in my opinion.

However, if this is a strong preference, I can make the change.